std_msgs/Header        header
TrafficLight[]         lights

